applications/StructuralMechanicsApplication/tests/shell_test/Shell_Q4_Thin_linear_dynamic_test_parameters.json

```diff
@@ -0,0 +1,139 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "Shell_Q4_Thin_linear_dynamic_test",
+        "model_part_name" : "Structure",
+        "domain_size"     : 3,
+        "time_step"       : 1.125,
+        "start_time"      : 0.0,
+        "end_time"        : 25,
+        "echo_level"      : 0
+    },
+        "material_import_settings" :{
+            "materials_filename": "shell_test/Shell_Q4_Thin_linear_dynamic_test_materials.json"
+        },
+    "solver_settings"          : {
+        "solver_type"                        : "structural_mechanics_implicit_dynamic_solver",
+        "echo_level"                         : 0,
+        "solution_type"                      : "Dynamic",
+        "time_integration_method"            : "Implicit",
+        "scheme_type"                        : "Newmark",
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "shell_test/Shell_Q4_Thin_linear_dynamic_test"
+        },
+        "material_import_settings" :{
+            "materials_filename": "shell_test/Shell_Q4_Thin_linear_dynamic_test_materials.json"
+        },
+        "line_search"                        : false,
+        "convergence_criterion"              : "Residual_criterion",
+        "displacement_relative_tolerance"    : 1e-9,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 1e-9,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 500,
+        "linear_solver_settings"             : {
+            "solver_type" : "Super_LU",
+            "scaling"     : false,
+            "verbosity"   : 0
+        },
+        "problem_domain_sub_model_part_list" : ["Parts_surface"],
+        "processes_sub_model_part_list"      : ["DISPLACEMENT_edge","DISPLACEMENT_sample","ROTATION_edge","SurfaceLoad3D_surface"],
+        "rotation_dofs"                      : true
+    },
+    "constraints_process_list" : [{
+        "python_module"   : "impose_vector_value_by_components_process",
+        "kratos_module" : "KratosMultiphysics",
+        "help"                  : "This process fixes the selected components of a given vector variable",
+        "process_name"          : "ImposeVectorValueByComponentsProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "DISPLACEMENT_edge",
+            "variable_name"   : "DISPLACEMENT",
+            "is_fixed_x"      : true,
+            "is_fixed_y"      : true,
+            "is_fixed_z"      : true,
+            "value"           : [0.0,0.0,0.0]
+        }
+    },{
+        "python_module"   : "impose_vector_value_by_components_process",
```

Comment:
This process is deprecated. @rubenzorrillcould u point to the list of correct processes?

Reply:
ok thx
This should work @rubenzorrilla

Reply:
@rubenzorrilla I am merging this branch now, it has been a while
I will correct the processes in a separate commit, so please let me know which process to use (I couldn't figure it out myself)